spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -884,8 +885,9 @@ def process_signatures(self, params):
                         parameter['label']))
                 tooltip_text = "{0}{1}".format(parameter_str, func_doc)
                 self.show_calltip(
-                    title, tooltip_text, color='#999999')
-        except Exception:
+                    title, tooltip_text, color='#999999', signature=True)
+        except Exception as e:
+            print(e)

```

Comment:
Why we do not use the error as part of the logging?

Reply:
See the definition of `log_lsp_handle_errors` above.

Reply:
But it seems on Py2 we completely loose the error, cause I did not know what was happening until I printed the actual exception. All I was getting was an error on the logger and not the actual error.

Reply:
Yeah, that needs to be fixed (I didn't test things in Python 2). Please open a new PR about it.